lib/net/validations.rb

```diff
@@ -3,6 +3,9 @@
 
 module Net
   module Validations
+    RECORD_UNIQUE_PRESENCE = N_("%{field} must be present and unique for %{model}")
+    RECORD_UNIQUE = N_("%{field} must be unique for %{model}")
```

Comment:
The field and models won't be translated, I think it may be better to just use the specific message for every validation instead of a constant string with interpolation